src/pages/Dashboard/Investment.tsx

```
@@ -0,0 +1,69 @@
+      completionDate: "2023-05-01",
+      status: "Pending"
+    }
+  ]
+    
+ 
+  return (
+    <div>
+        <h1 className="text-[28px] mb-6">Investments</h1>
+
+        <Table data={tableData} tableHead={tableHead}>
+          {tableData.map((table: any, index: number) => (
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)